src/mixins/filing-template-mixin.ts

```diff
@@ -587,28 +601,20 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     }
     this.setOfficeAddresses(addresses || entitySnapshot.addresses)
 
+    let parties = filing.changeOfRegistration.parties || entitySnapshot.orgPersons
```

Comment:
Can `orgPersons` be null or undefined? If so the line 608 would break.

Reply:
I don't believe `parties` could be defined as `{}`, no. 
It's either defined in the Filing because it was modified and added, or the snapshot will provide the data. If the snapshot failed to get the information it is handled in `fetchFirmSnapshot()`

Reply:
If it's not in the filing then it's undefined or null? That works.

Reply:
You bet, most parts of the change filing can be null or undefined.

Reply:
Thanks for checking. Resolving.